lib/DraggableCore.es6

```diff
@@ -319,18 +320,31 @@ export default class DraggableCore extends React.Component {
     return this.handleDragStop(e);
   };
 
+  removeScroll: EventHandler<MouseEvent> =  (e) => {
+    e.preventDefault();
+    return false;
+  };
+
   // Same as onMouseDown (start drag), but now consider this a touch device.
   onTouchStart: EventHandler<MouseEvent> = (e) => {
     // We're on a touch device now, so change the event handlers
     dragEventFor = eventsFor.touch;
 
+    // Stop the touch device from scrolling while user is dragging.
+    if (this.state.dragging) {
+      document.addEventListener('touchmove', this.removeScroll)
```

Comment:
It is on the document to stop the document from scrolling on touch. I want the draggable to respond to touchmove, not the page.